assembl/static2/js/app/components/administration/landingPage/manageModules.jsx

```diff
@@ -36,33 +35,55 @@ export const DumbManageModules = ({
   moveModuleUp,
   toggleModule,
   createModule
-}: Props) => (
-  <div className="admin-box">
-    <SectionTitle
-      title={I18n.t('administration.landingPage.manageModules.title')}
-      annotation={I18n.t('administration.annotation')}
-    />
-    <div className="admin-content form-container" style={{ maxWidth: '700px' }}>
-      <p className="admin-paragraph">
-        <Translate value="administration.landingPage.manageModules.helper" />
-      </p>
-      <div className="two-columns-admin">
-        <div className="column-left">
-          <SelectModulesForm lang={locale} moduleTypes={moduleTypes} modulesById={modulesById} toggleModule={toggleModule} />
-          <div className="margin-xl">
-            <Button className="button-submit button-dark" onClick={() => createModule(enabledModules.size - 2)}>
-              <Translate value="administration.landingPage.manageModules.textAndMultimediaBtn" />
-            </Button>
+}: Props) => {
```

Comment:
Recently, after speaking with @pascalhuynh regarding making components easier to create/mock. As is, this component's `submitButton` is baked in and cannot be changed. So perhaps you could pull the `displayConfirmationModal` out of the definition of the component, and pass it in as a prop, like `handleSubmitButton`. And the `displayConfirmationModal` method can take the component props as input and pick up what it needs internally. It's a better way to mock/test these components in future.

Reply:
I think that it's a specific business component and not abstract (ui or business component). We don't need to reuse it. in my opinion the solution is sufficient.